bucket/tcno-acc-switcher.json

```diff
@@ -0,0 +1,70 @@
+{
+    "version": "3.0.02",
+    "description": "A super fast account switcher for Steam.",
+    "homepage": "https://github.com/TcNobo/TcNo-Acc-Switcher",
+    "license": "GPL-3.0-only,LGPL-2.1-only,...",
+    "notes": ".NET Framework 4.8 is required.",
+
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TcNobo/TcNo-Acc-Switcher/releases/download/3.0.02/TcNo.Account.Switcher.x64.zip",
+            "hash": "42d17824569d311451e78cf5f12333f1c6af2dc9080b68c5226c741e4e2aa1e4",
+            "extract_dir": "TcNo.Account.Switcher.x64",
+            "pre_install": [
+                "# Create placeholder files",
+                "function CreateFile([String[]] $file) {",
+                "    foreach ($f in $file) {",
+                "        if (! (Test-Path \"$dir\\$f\")) { New-Item \"$dir\\$f\" -ItemType 'File' | Out-Null }",
+                "    }",
+                "}",

```

Comment:
- Why this file is created only in 64bit??
- Full function for creating one file is completely nonsense...

Reply:
Why are you reviewing this when it's clearly incomplete?

Reply:
Why do you open PR when it is clearly incomplete?

Reply:
I was being lazy and hoped someone would do the work for me.